src/main/java/edu/cornell/kfs/gl/service/ScheduledAccountingLineService.java

```diff
@@ -0,0 +1,13 @@
+package edu.cornell.kfs.gl.service;
+
+import java.sql.Date;
+import java.util.TreeMap;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.fp.businessobject.ScheduledSourceAccountingLine;
+
+public interface ScheduledAccountingLineService {
+	TreeMap<Date, KualiDecimal> generateDatesAndAmounts(ScheduledSourceAccountingLine scheduledAccountingLine, int rowId);
```

Comment:
these are places where javadoc comments should probably be added since it's a public service